filecache.go

```diff
@@ -198,20 +220,20 @@ func (c *FileCache) GetFileName(filename string) string {
 	extension := c.DefaultExtension
 
 	// Look in the last 5 characters for a . and extension
-	lastDot := strings.LastIndexByte(filename, '.')
-	if lastDot > len(filename)-6 {
-		extension = filename[lastDot:]
+	lastDot := strings.LastIndexByte(file.Path, '.')
+	if lastDot > len(file.Path)-6 {
+		extension = file.Path[lastDot:]
 	}
 
-	file := fmt.Sprintf("%x%s", hashedFilename, extension)
+	fileName := fmt.Sprintf("%x%s", hashedFilePath, extension)
 	dir := fmt.Sprintf("%x", hashedDir[:1])
-	return filepath.Join(c.BaseDir, dir, filepath.FromSlash(path.Clean("/"+file)))
+	return filepath.Join(c.BaseDir, dir, filepath.FromSlash(path.Clean("/"+fileName)))
 }
 
 // onEvictDelete is a callback that is triggered when the LRU cache expires an
 // entry.
 func (c *FileCache) onEvictDelete(key interface{}, value interface{}) {
-	filename := key.(string)
```

Comment:
This change actually makes this harder to understand. It's not a file, it's a filename, right?

Reply:
Well, it's a file path, so it confused me too. I don't have a good idea about how to name it, so I can leave it as it was if you think that's clearer @relistan.